nx-X11/programs/Xserver/hw/nxagent/Window.c

```diff
@@ -466,6 +466,18 @@ FIXME: Do all the windows for which nxagentWindowTopLevel(pWin)
     nxagentRedirectWindow(pWin);
   }
 
+  if ((nxagentRealWindowProp) && (!nxagentWindowTopLevel(pWin)))
+  {
+    Atom prop = MakeAtom("NX_REAL_WINDOW", strlen("NX_REAL_WINDOW"), True);
```

Comment:
Yeah, I did not mean that from a function POV. I simply think having all the atoms we use at one place makes sense.